pkg/operator/staticpod/startupmonitor/cmd.go

```diff
@@ -79,11 +83,12 @@ func NewCommand(check ReadinessChecker) *cobra.Command {
 				withProbeInterval(time.Second).
 				withTimeout(o.FallbackTimeout)
 
-			fb := newFallback().
+			fb := newStaticPodFallback().
 				withRevision(o.Revision).
 				withManifestPath(o.ManifestDir).
 				withStaticPodResourcesPath(o.ResourceDir).
-				withTargetName(o.TargetName)
+				withTargetName(o.TargetName).
+				withNodeName(o.NodeName)
 
 			if len(o.KubeConfig) > 0 {
```

Comment:
since `KubeConfig` is required we don't need that `if` clause anymore

Reply:
fixed

Reply:
But there could be a in-cluster config. Either has to work. So kubeconfig is not required if in-cluster config is provided.